.github/workflows/deploy_staging_firebase.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
       with:
-        fetch-depth: 0
```

Comment:
The indent is changed 🤔

Reply:
@markgravity You did not check this

Reply:
@suho The indent is changed to correct the indent 🙏